$OpenBSD: patch-testsuite_libffi_call_float2_c,v 1.2 2010/09/19 20:19:15 jasper Exp $

The %Lf printf format expects "long double", so cast args accordingly.

Patch from NetBSD.

--- testsuite/libffi.call/float2.c.orig	Tue Dec 29 16:22:26 2009
+++ testsuite/libffi.call/float2.c	Sun Sep 19 21:32:27 2010
@@ -45,7 +45,7 @@ int main (void)
   /* This is ifdef'd out for now. long double support under SunOS/gcc
      is pretty much non-existent.  You'll get the odd bus error in library
      routines like printf().  */
-  printf ("%Lf, %Lf, %Lf, %Lf\n", ld, ldblit(f), ld - ldblit(f), LDBL_EPSILON);
+  printf ("%Lf, %Lf, %Lf, %Lf\n", (long double)ld, (long double)ldblit(f), (long double)ld - ldblit(f), (long double)LDBL_EPSILON);
 #endif
 
   /* These are not always the same!! Check for a reasonable delta */
